projects/fal/src/fal/api.py

```diff
@@ -23,10 +23,17 @@
 
 import dill
 import dill.detect
-import fal.flags as flags
 import grpc
 import isolate
 import yaml
+from isolate.backends.common import active_python
```

Comment:
This was sorted automatically with ruff. I can undo this if we want to group isolate modules with fal modules.